.circleci/config.yml

```diff
@@ -55,7 +55,8 @@ jobs:
   deploy:
     <<: *defaults
     steps:
-      - checkout
+      - attach_workspace:
+          at: .
```

Comment:
I was trying to figure out how this ever worked before... For anyone else who is curious, it turns out that this step used to do `attach_workspace`, but it accidentally got dropped: https://github.com/gruntwork-io/cloud-nuke/commit/cc11314d70f2632d2c86e405c1b02dcfaf9c6fb4#diff-1d37e48f9ceff6d8030570cd36286a61L30-L31